tensornetwork/ncon_interface.py

```diff
@@ -147,6 +176,12 @@ def _build_network(tensors, network):
     nodes = []
     edges = {}
     for (i, (tensor, edge_lbls)) in enumerate(zip(tensors, network)):
+        # Allow single indices to be passed as scalars.
+        try:
+            edge_lbls = list(edge_lbls)
```

Comment:
This is gonna cause some issues when edge_lbls are strings.

Reply:
Ah, good point.